subgraphs/uniswap-forks/deployment/deploymentConfigurations.json

```diff
@@ -81,6 +81,13 @@
         "steegecs": "steegecs/sushiswap-xdai",
         "messari": "messari/sushiswap-xdai"
       }
+    },
+    "spookyswap": {
+      "fantom": {
+        "template": "spookyswap.template.yaml",
+        "test": "dhruvjain1122/spookyswap_messari",
+        "messari": "messari/spookyswap"
```

Comment:
The naming convention for the messari hosted service is -> messari/{protocol}-{network}. So in this case, it should be messari/spookyswap-fantom.